main/main.cpp

```diff
@@ -2926,7 +2925,8 @@ Error Main::setup2(bool p_show_boot_logo) {
 
 		Input *id = Input::get_singleton();
 		if (id) {
-			agile_input_event_flushing = GLOBAL_DEF("input_devices/buffering/agile_event_flushing", false);
+			bool agile_input_event_flushing = GLOBAL_DEF("input_devices/buffering/agile_event_flushing", false);
```

Comment:
This setting (like input accumulation) is no longer documented in the ProjectSettings class reference, but it still exists.

Reply:
> Also, is there much of a point in having editor settings for agile event flushing and input accumulation? I can't see much reasons for changing their value from the default. 

For both settings, yes there's value. The editor being itself a Godot Android app, and being expected to run onto a wide variety of Android devices, this allows users to select the options best suited to their devices regardless of the project specific settings. 
And to us devs, this allows to quickly validate any input related changes with either or both of those settings enabled / disabled. For example, I used those editor settings to test the changes in https://github.com/godotengine/godot/pull/93933.

Reply:
> This setting (like input accumulation) is no longer documented in the ProjectSettings class reference, but it still exists.

It's still documented. `use_input_buffering` and `use_accumulated_input` were the ones removed. Both were introduced in https://github.com/godotengine/godot/pull/93933:
- `use_input_buffering` was deprecated in favor or `agile_event_flushing`
- the `use_accumulated_input` project setting was added in error. I overlooked that the setting is already exposed via [`Input::use_accumulated_input`](https://docs.godotengine.org/en/stable/classes/class_input.html#class-input-property-use-accumulated-input).

Reply:
> the use_accumulated_input project setting was added in error. I overlooked that the setting is already exposed via [Input::use_accumulated_input](https://docs.godotengine.org/en/stable/classes/class_input.html#class-input-property-use-accumulated-input).

It's indeed already exposed via the Input singleton, but the project setting was added so that you can disable it without needing to use an autoloaded script or something similar. It's similar to project settings like `display/window/size/transparent`.

Reply:
> It's indeed already exposed via the Input singleton, but the project setting was added so that you can disable it without needing to use an autoloaded script or something similar. It's similar to project settings like `display/window/size/transparent`.

The project setting was only in https://github.com/godotengine/godot/pull/93933/files#diff-18b7c19417a461e4793bb2157831659d96ede991391e10ee5bc51f19453b7685, and that was mostly because I forgot about the `Input` singleton property. You can check the code prior to that, there's no reference to a project settings for `use_accumulated_input`.